src/components/common/Extensions/ExtensionUpgradeButton/ExtensionUpgradeButton.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useCallback } from 'react';
+
+import {
+  Extension,
+  ExtensionVersion,
+  isExtensionCompatible,
+} from '@colony/colony-js';
+import { ActionButton } from '~shared/Button';
+import { AnyExtensionData } from '~types';
+import { ActionTypes } from '~redux/index';
+import { mapPayload } from '~utils/actions';
+import { useAppContext, useColonyContext } from '~hooks';
+
+interface Props {
+  extensionData: AnyExtensionData;
+}
+
+const ExtensionUpgradeButton = ({ extensionData }: Props) => {
+  const { colony } = useColonyContext();
+  const { user } = useAppContext();
+
+  const transform = useCallback(
+    mapPayload(() => ({
+      colonyAddress: colony?.colonyAddress,
+      extensionId: extensionData.extensionId,
+      version: extensionData.availableVersion + 1,
```

Comment:
`availableVersion` means the highest version that is available in the network. I think you want to use `extensionData.version` which indicates the currently installed version. My bad, I see how this can be confusing so please feel free to rename the `version` field on the `Extension` GraphQL fragment to something like `currentVersion` if you wish
Please note this field won't be accessible unless you change the `extensionData` prop type to `InstalledExtensionData` on line 15